ontologies/CALM_single_MS.md

```diff
@@ -0,0 +1,24 @@
+
+# JSON examples: a free-standing manuscript described in CALM
+
+Comment, lines 8-13
+CALM contains both RefNo (to build the hierarchy) and AltRefNo (to show to the user).
+Both will presumably be needed.  At present each maps simply to identifier (=sourceIdentifier in the .ttl files).  
+How to distinguish?
+Also occurs with Item, below
+
+Comment, lines 47-52
+CALM contains both RefNo (to build the hierarchy) and AltRefNo (to show to the user).
+Both will presumably be needed.  At present each maps simply to identifier (=sourceIdentifier in the .ttl files).  
+How to distinguish?
+Also occurs with work, above
+
+Comment, line 55
+I've concatenated here two fields from CALM.
+"Extent" is a very top-level indication of how much stuff there is - "1 file"
+"Physical Description" goes into more detail.
+Do we want to separate these, as there's a fair bit of real-world overlap in how they're used?
+
+Comment, line 37
+Proved necessary to replace double quotes around the article title with single ones in order for JSON to validate
+Is this strictly necessary?  Ideally double quotes would remain: consistent with scholarly style guides, etc.
```

Comment:
Good question! You can *escape* double quotes in JSON by preceding them with a backslash – the backslash tells a JSON interpreter to treat the next character as a literal character, rather than a closing quote of a JSON string. For example:

```json
{
    "name": "He said \"My name is Bond. James Bond.\" The smile told Bond that he was expected."
}
```